playbooks/gcp/openshift-cluster/install.yml

```diff
@@ -24,12 +24,3 @@
 
 - name: install components
   import_playbook: ../../common/private/components.yml
-
-- hosts: primary_master
-  gather_facts: no
-  tasks:
-  - name: Retrieve cluster configuration
```

Comment:
What is the purpose of this task?

Reply:
Ha, looks like this is used by CI to see if a container is up.  Thanks @vrutkovs

Reply:
Ah, yes, this is required to make `test` container proceed.

This however needs to be moved in a separate ansible call for CI, smth like https://github.com/vrutkovs/release/commit/f25995cb80f66066bb2b0e3681018f388560e274#diff-79defd8db05af250d42aa465c7284a7aR220